codeToggles.js

```diff
@@ -1,7 +1,10 @@
 // This file is a map of github issue numbers with booleans
 // TODO: move to a modifiable location such as S3 or DynamoDB
 const toggles = {
+  6506: true,
+  6841: true,
   6868: true,
+  6915: true,
```

Comment:
I'm confused about this issue being set to true when there's no toggle in the code right now. Is the related code in the right state right now?

Reply:
My bad, there *is* a toggle for this, just not where I was looking previously.